src/makeCalculator.js

```diff
@@ -37,7 +37,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  const calculator = {
+    add: (number) => {
+      result += number;
+
+      return calculator;
+    },
+    subtract: (number) => {
+      result -= number;
+
+      return calculator;
+    },
+    multiply: (number) => {
+      result *= number;
+
+      return calculator;
+    },
+    divide: (number) => {
+      result /= number;
```

Comment:
There is no check for zero before division. This might lead to a 'Division by Zero' error.